node/src/service.rs

```diff
@@ -67,6 +66,15 @@ use sp_transaction_pool::runtime_api::TaggedTransactionQueue;
 use std::sync::Arc;
 use substrate_prometheus_endpoint::Registry;
 
+#[cfg(not(feature = "runtime-benchmarks"))]
+type HostFunctions = sp_io::SubstrateHostFunctions;
+
+#[cfg(feature = "runtime-benchmarks")]
+type HostFunctions = (
+    sp_io::SubstrateHostFunctions,
+    frame_benchmarking::benchmarking::HostFunctions,
+);
+
 /// Native Manta Parachain executor instance.
 pub struct MantaRuntimeExecutor;
```

Comment:
Seems it's removable? I didn't any place where it's being used.

Reply:
Nope.  It's used in commands.rs in benchmark and try-runtime mode.